mobile/lib/shared/models/asset.dart

```diff
@@ -108,6 +110,8 @@ class Asset {
   )
   String checksum;
 
+  String? thumbhash;
```

Comment:
Please store it as `List<byte>` instead, this is faster and uses less space.

Reply:
The asset response from the server is a string.

Reply:
Also the checksum (which is basically the exact same format as thumbhash) is stored as string with this comment. 

```dart
  /// stores the raw SHA1 bytes as a base64 String
  /// because Isar cannot sort lists of byte arrays```

Reply:
I put the comment there :-)

Please decode the base64 string to bytes and store it in the DB (we don't ever need to sort based on thumb hash).

The thumb hash library has a constructor fromBytes and fromIntList that could both be used instead of fromBase64.

Makes the thumb hash image generation even faster because base64 decode is no longer needed every time.

Thank you for your help !

Reply:
yeah. it's a Uint8List in Dart under the hood. not sure why they did not reuse Uint8List.. who knows :-)